docs-guides/source/mlmodel-utilities.md

```diff
@@ -172,3 +172,47 @@ config = cto.coreml.OptimizationConfig(
 compressed_mlmodel = cto.coreml.palettize_weights(mlmodel, config)
 
 ```
+
+## Bisect Model
+
+When working with large Core ML models, for instance a big [Stable Diffusion](https://github.com/apple/ml-stable-diffusion) model, the process might be killed due to memory issues. You can solve the issue by chunking the previously exported model into two pieces, and combine them into a Core ML pipeline model.
+
+The example below shows how to bisect a model, test the accuracy, and savethem on disk.
+
+```python
+import coremltools as ct
+
+model_path = "my_model.mlpackage"
+output_dir = "./output/"
+
+# The following code will produce two chunks models:
```

Comment:
"The following code will produce two chunks models:" -- > "The following code will produce two smaller models"

Reply:
Done!